test/dummy/db/schema.rb

```diff
@@ -11,42 +10,41 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150112232813) do
+ActiveRecord::Schema.define(version: 2015_12_10_120915) do
```

Comment:
BTW, once the tests are fixed, can you get rid of these changes as well as the new files (test/dummy/config/database_dev.yml, test/dummy/config/database_github.yml, etc.), which shouldn't be needed?

Would also be great if you could clean-up the Git history (new PR might be easiest).

Thanks!

Reply:
> BTW, once the tests are fixed, can you get rid of these changes as well as the new files (test/dummy/config/database_dev.yml, test/dummy/config/database_github.yml, etc.), which shouldn't be needed?
> 
> Would also be great if you could clean-up the Git history (new PR might be easiest).
> 
> Thanks!

Will be done.

Reply:
killbill-client-ruby is not active record model, so ActiveModel::Validation will not work.

Reply:
Running tests